### DISCUSSION TITLE
Request for GitHub teacher benefits is Pending Indefinatly

### DISCUSSION THREAD

**Discussion by DoctorHayes** [community/community#17706] [GitHub Education]
I meet all of the criteria for teacher benefits and submitted my request on Jan 1, 2022 (6 months ago). My request is still pending. Any idea how long this should take?

I am getting emails that my coupon is about to expire and that if I am a teacher I should apply for the academic discount. However, I have obviously already done that (6 months ago). I am hoping my request is processed before my coupon expires.

Any suggestions? Should I cancel my request and resubmit it? I don’t want to be put at the end of the queue and wait another 6 months.

**Followup:**  
Please help. It has now been over 9 months and my application for Teacher benefits is still pending! If I need to submit more information or start a new application I am willing.

It seems that GitHub has the worst customer service of any company I have ever worked with. I have never had to wait so long for a response and there doesn't appear to be any way to contact GitHubQ to get this fixed. It is hard to recommend using GitHub to my students.

Since GitHub isn't listening, I would appreciate it if any other teachers have any suggestions. I am all ears!

**Comment by alexandercbooth-test** [comment]
t
